src/site/content/en/blog/get-installed-related-apps/index.md

```diff
@@ -181,23 +181,29 @@ the full path to the PWAs web app manifest in the `url` property.
 }
 ```
 
-#### PWA served from a different domain
+#### PWA served from a different location
 
 {% Aside %}
 **Coming Soon!**
-In addition to detecting if their PWA is served from the same domain,
-Chrome 82 lets websites check if their PWA is served from a different domain.
-Microsoft is actively working on enabling this API for Edge and we hope to
-see it land around Edge 82.
+Starting in Chrome 82, a page can check if its PWA is installed, even if it
+is outside the scope of the PWA. Microsoft is actively working on enabling
+this API for Edge and we hope to see it land around Edge 82.
 {% endAside %}
 
-If the PWA you're testing for lives on a different domain, for instance
-`example.com` wants to check if the PWA served from `app.example.com` is
-installed, you'll also need to define the relationship between the two sites.
+A page can check if its PWA is installed, even if it is outside the
+[scope][scope] of the PWA. For instance, `www.example.com` wants to check if
+its PWA served from `app.example.com` is installed. Or a page under
+`/about/` to check if the PWA served from `/app/` is already installed.
+
+First, add a web app manifest that includes the `related_applications` property
+([as indicated above](#relationship-web)) to the page that will check if it's
```

Comment:
```suggestion
([as indicated above](#relationship-web)) to the page that will check if its
```

Reply:
I f'ing hate apostrophes! I thought I got them all right this time. :) Oh well.

Reply:
There's a simple trick. Say or think the expanded contraction in the whole sentence. If "it is" sounds wrong, you don't need the apostrophe.